src/frformat/enum_format.py

```diff
@@ -11,20 +11,22 @@ class EnumFormat(CustomStrFormat):
 
         May preprocess the input and valid values according to given "options"."""
 
-        def __init__(self, options: Options):
-            self._options = options
+        def __init__(self, options: Optional[Options] = None):
+            self.options = options if options is not None else Options()
```

Comment:
```suggestion
        def __init__(self, options: Options = Options()):
            self.options = options
```

Petite préférence pour l'attribut privé plutôt que public.